### PR TITLE
Some small enhancements for tab-focus and around

### PR DESCRIPTION
**Pull request by ymo-sci**
### Additional Notes

- This PR fixes or works on following ticket(s): [SIRI-900](https://scireum.myjetbrains.com/youtrack/issue/SIRI-900)
- This PR is related to PR: https://github.com/scireum/sirius-web/pull/1315 and https://github.com/scireum/token-autocomplete/pull/69

### Checklist

- [x] Code change has been tested and works locally
- [x] Code was formatted via IntelliJ and follows SonarLint & [best practices](https://scireum.myjetbrains.com/youtrack/articles/MISC-A-16/CodeStyle-JavaDoc)
- [ ] Patch Tasks: Is local execution of Patch Tasks necessary? If so, please also mark the PR with the tag.

### CODE REVIEW

File: src/main/resources/default/taglib/t/lookupValue.html.pasta [hunk 1]
```diff
@@ -21,21 +21,27 @@
...
-                new LookupTableInfo({
-                    tableName: '___table',
-                    labelFormat: '___value.getDisplay()',
-                    codeCallback: function (code, name) {
-                        autocomplete.val({value: code, text: name});
-                    }
-                });
+            helper.addEventListener('click', getLookupTableInfo);
+            helper.addEventListener('keydown', function (e) {
+                if (e.keyCode === 13) {
```
Comment (mko-sci) [inline src/main/resources/default/taglib/t/lookupValue.html.pasta:32]: Maybe but the 13 in a variable?

**Review by mko-sci — APPROVED** [review]

**Review by idlira — APPROVED** [review]
Don't you need to bring the updated token-autocomplete over here too? Or is it only embedded in the final products (although sirius-biz can start standalone)

File: src/main/resources/default/taglib/t/lookupValue.html.pasta (hunk 1 above)
Comment (sabieber) [reply src/main/resources/default/taglib/t/lookupValue.html.pasta:32]: Mhhh if we do so I would put a KeyCodes list in sirius, like `sirius.KeyCode.ENTER`

**Comment by sabieber** [comment]
> Don't you need to bring the updated token-autocomplete over here too? Or is it only embedded in the final products (although sirius-biz can start standalone)

TokenAutocomplete is only included in web :)

**Review by sabieber — APPROVED** [review]

**Review by mkeckmkeck — APPROVED** [review]

File: src/main/resources/default/taglib/t/lookupValue.html.pasta (hunk 1 above)
Comment (ymo-sci) [reply src/main/resources/default/taglib/t/lookupValue.html.pasta:32]: 👍 will follow in an extra PR after https://github.com/scireum/sirius-web/pull/1317 is merged.

**Review by fhaScireum — APPROVED** [review]

File: src/main/resources/default/taglib/t/lookupValue.html.pasta (hunk 1 above)
Comment (jakobvogel) [reply src/main/resources/default/taglib/t/lookupValue.html.pasta:32]: https://github.com/scireum/sirius-web/pull/1317 is merged now, the respective version of web is available in `develop`. Please merge `develop` into this PR's feature branch and modify this line accordingly.

**Review by jakobvogel — APPROVED** [review]

**Review by fhaScireum — APPROVED** [review]

**Pull request merged**